recipes/guessmylt/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "guessmylt" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ce97be361e0fbfa0409fbce85d2d31b56cc05b52b4fea23492b184ac92c9e9ca
+
+build:
+  number: 0
+  noarch: python
+  skip: True # [osx]
+  entry_points:
+    - GUESSmyLT = GUESSmyLT.GUESSmyLT:main
+    - GUESSmyLT-example = GUESSmyLT.test.example:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python>3
+    - bcbio-gff ==0.6.4
+    - biopython ==1.67
+    - pip
+    - pysam >=0.13.0
+    - snakemake ==5.4.*
+    - trinity ==2.8.4
+    - busco ==3.0.2
+    - bowtie2 >=2
+  run:
+    - python>3
+    - git >=2.11.1
+    - bcbio-gff ==0.6.4
+    - biopython ==1.67
+    - pysam >=0.13.0
+    - snakemake ==5.4.*
+    - trinity ==2.8.4
+    - busco ==3.0.2
+    - bowtie2 >=2
+
+test:
+  imports:
+    - GUESSmyLT
+  commands:
+    - GUESSmyLT --help
+    - GUESSmyLT-example
+
+about:
+  home: https://github.com/NBISweden/GUESSmyLT
+  license: GPLv3
+  license_family: GPL
+  license_file: LICENSE
+  summary: Software to guess the RNA-Seq library type.
+  dev_url: https://github.com/NBISweden/GUESSmyLT
+
+extra:
+  recipe-maintainers:
+    - juke34
+  skip-lints: #  The skip mechanism does not work in conjunction with noarch but I need to skip osx due to inconsistency in BUSCO
```

Comment:
As mentioned above, you cannot have both noarch and skip. It just doesn't work. This isn't the linter, this is conda. If your package will not work on OSX, you need to make it `noarch:false` (or remove line) and add `- should_be_noarch  # will not work on OSX due to BUSCO ...`.

Reply:
Thx for the explanation. I understood how it works. Should be fine now.